spec/skopeo_spec.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env shellspec
 
 Describe "printVersion()"
-  printVersion() {
-    skopeo -v | grep "${SKOPEO_VERSION}"
-  }
+printVersion() {
+  skopeo -v | grep "${SKOPEO_BINARY_VERSION}"
+}
 
-  It "validates tool is installed by checking version"
-    When call printVersion
-    The output should include "${SKOPEO_VERSION}"
-    The status should eq 0
-  End
+It "validates tool is installed by checking version"
```

Comment:
One last thing

This file needs to be indented like the other `shellspec` files